src/components/lyrics.tsx

```diff
@@ -0,0 +1,74 @@
+import { useEffect, useState } from "react";
+
+export default function Lyrics({ lyricsLink }: { lyricsLink: string }) {
+  const [translatedLyrics, setTranslatedLyrics] = useState<string | null>(null);
+  const [lyrics, setLyrics] = useState<string | null>(null);
+
+  const fetchLyrics = async () => {

```

Comment:
move these functions(fetchLyrics and translateLyrics) outside the component, return fetch result fom they. then use this result to further actions